modules/effects/src/main/scala/dev/profunktor/redis4cats/interpreter/Redis.scala

```diff
@@ -96,45 +129,45 @@ private[redis4cats] class BaseRedis[F[_]: ContextShift, K, V](
 
   import scala.collection.JavaConverters._
 
-  private val async: F[RedisClusterAsyncCommands[K, V]] =
+  def async: F[RedisClusterAsyncCommands[K, V]] =
     if (cluster) conn.clusterAsync else conn.async.widen[RedisClusterAsyncCommands[K, V]]
 
-  override def del(key: K*): F[Unit] =
+  def del(key: K*): F[Unit] =
     JRFuture {
       async.flatMap(c => F.delay(c.del(key: _*)))
     }.void
 
-  override def expire(key: K, expiresIn: FiniteDuration): F[Unit] =
+  def expire(key: K, expiresIn: FiniteDuration): F[Unit] =
     JRFuture {
       async.flatMap(c => F.delay(c.expire(key, expiresIn.toSeconds)))
     }.void
 
   /******************************* Transactions API **********************************/
   // When in a cluster, transactions should run against a single node, therefore we use `conn.async` instead of `conn.clusterAsync`.
 
-  override def multi: F[Unit] =
+  def multi: F[Unit] =
     JRFuture {
-      conn.async.flatMap(c => F.delay(c.multi()))
+      async.flatMap(c => F.delay(c.asInstanceOf[RedisAsyncCommands[K, V]].multi()))
```

Comment:
I don't like this hack here. I'll see if I can come up with a better way to handle this.